### ISSUE TITLE
Feature Request: Analysis Queue

### ISSUE THREAD

**Issue by destinrl** [cirquit/trdrop#73]
Say you have 5 videos captured and you'd like the tool to analyze them one at a time individually. It would be amazing if you could setup TR drop to simply go through them one at a time with pre-set settings. Thing Adobe Media Queue.

**Comment by cirquit** [comment]
That is a very interesting proposal. The problems I see with it right now is the possibility to analyze up to 3 videos in parallel, so this "option", as well as their order has to be set beforehand. I'm assuming all the settings should be copied over?

Another issue is the naming of the projects, because you'd have to provide a name for each one individually, as well as start exporting automatically after the previous one is finished if I understand correctly.

This would be a very big endeavor, because I currently lack the functionality and the visual representation (think a specific popup window) to have enable this batch processing.

How would you use this feature or why do you want to have it? Just so I can a general idea of the scope and may propose other solutions.

**Comment by destinrl** [comment]
For me I like to analyze PS5, then Xbox Series X, the Xbox Series S and then when editing how I want it to look in premiere I will move the FPS marker, add an appropriate logo (the ability to add an icon would be a nice feature btw)

Sometimes I will run the analysis side-by-side, but I prefer to do that manually in premiere right now.

**Comment by cirquit** [comment]
So let me understand it better - you want 3 videos from different consoles (probabaly the same game) to be processed with the **same** settings one after the other? 

Do you change the color or is it the same for all of them? Do you remove the framerate plot as well or how do you crop them together in premiere? There should be strange overlaps happening, both with the fps text as well as with the plots if you do it that way.

Can you maybe share a screenshot of a finished video layout in premiere?

The logo stuff might be easy enough to do, but can you maybe provide an example for that as well?
